tools/tertiary-analysis/scmap/scmap_get_std_output.xml

```diff
@@ -5,11 +5,13 @@
     </macros>
     <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
-        scmap_get_std_output.R --predictions-file "${input_predictions_file}" --output-table "${output_predictions_file}" --include-scores "${include_scores}" --sim-col-name "${sim_col_name}"
+        scmap_get_std_output.R --predictions-file "${input_predictions_file}" --output-table "${output_predictions_file}" --include-scores "${include_scores}" --index '${index}' --tool '${tool}' --sim-col-name "${sim_col_name}"
     ]]></command>
     <inputs>
         <param type="data" name="input_predictions_file" label="Scmap predictions file in text format" format="txt" help="Path to the predictions file in text format" />
         <param type="boolean" name="include_scores" checked="false"  label="Should prediction scores be included?" help="Boolean indicating whether similarity scores should be included in the final output" />
+        <param type="data" name="index" label="Scmap index object" format="rdata" help="Scmap index object in rdata format" />
+        <param type="text" name="tool" label="Tool used (cell/cluster)"  help="What tool was used - scmap-cell or scmap-cluster?" />
```

Comment:
If this is an options input, then it should be provided like that and not as a text input.